src/js/options.js

```diff
@@ -29,14 +29,24 @@ import ConfigFile from "./shared/config_file";
 import LocalStore from "./shared/local_store";
 import providerTabHelper from "./shared/provider_helper";
 
+// Inject Svelte components into the page.
+import Footer from "../components/options/footer.svelte";
+import Header from "../components/options/header.svelte";
+
+new Footer({
+  target: document.getElementById("footer"),
+});
+
+const myHeader = new Header({
+  target: document.getElementById("header"),
+});
+myHeader.$on("tabClicked", updateTabsVisibility);
```

Comment:
When you first open the Options page, the "Search Options" tab is selected, but the "Settings" tab is the one shown:

![image](https://user-images.githubusercontent.com/8558447/140516936-3f895f5e-ec64-40c6-bf36-8f161d0b45d2.png)

This probably happens because the `onMount` event is triggered _before_ you register the listener on `myHeader.$on`.

> I.e. dispatching events on `onMount` isn't a good idea.